### DISCUSSION TITLE
Line with Symbols

### DISCUSSION THREAD

**Discussion by Meibes** [mapsforge/vtm#1049] [Q&A] [answered]
Hello,

is it possible to draw a dotted line or a line consisting of triangles? something like this: -o-o-o-o- or that: -△-△-△-△-

i tried something like this, but it did not work:
```
<m e="way">
  <m k="highway">
    <symbol repeat="true" src="assets:symbols/dot_blue_dark.svg" />
  </m>
</m>
```

switching the `<symbol>` tag to a `<lineSymbol>` finally displayed something but it didn't look like what i was expecting, so i stopped there and decided to ask here

**Comment by devemux86** [comment, marked as answer]
One way is with symbol repeat like the oneways in the themes, see [example](https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/biker.xml#L1017).

Another way could be with line symbol, which creates a repeated texture.

**Reply by devemux86** [reply]
Also check the various parameters of these rules: "repeat-", billboard, rotation, etc.
